app/src/main/java/com/github/steroidteam/todolist/model/todo/Task.java

```diff
@@ -55,4 +57,12 @@ public boolean equals(Object o) {
         Task task = (Task) o;
         return Objects.equals(body, task.body);
     }
+
+    public Boolean isDone() {
+        return done;
+    }
+
+    public void setDone(Boolean done) {
```

Comment:
Same here, maybe "setIsDone" and "getIsDone" would be better for the boolean but maybe it's only a matter of personal taste haha